build/build_apple_frameworks.sh

```diff
@@ -211,7 +211,7 @@ sed -i '' '1i\
 sed -i '' '1i\
 #define C10_USING_CUSTOM_GENERATED_MACROS
 ' $HEADERS_PATH/executorch/runtime/core/portable_type/c10/macros/Export.h
-ln -s $HEADERS_PATH/executorch/runtime/core/portable_type/c10 "$HEADERS_PATH/"
```

Comment:
iirc, here's what worked for me:
`ln -s executorch/runtime/core/portable_type/c10 "$HEADERS_PATH/c10"`

Reply:
I would expect that we are likely to accidentally produce a framework containing a symbolic link. I would like to retreat to safety. I don't expect saving disk I/O for a couple dozen text files to be a significant win in 2025.